pages/jobs/index.tsx

```diff
@@ -41,110 +46,228 @@ const CountUp = ({startTime}) => {
 };
 
 export default function Jobs({items, project}) {
+	const [filterOpen, setFilterOpen] = useState(false);
+
+	const router = useRouter();
+
+	const currentRoute = router.asPath;
+	const basePath = router.pathname;
+
+	const isCompletedStateRoute = currentRoute.includes(
+		STATES.completedState.id.toString()
+	);
+	const isPendingStateRoute = currentRoute.includes(
+		STATES.pendingState.id.toString()
+	);
+	const isRunningStateRoute = currentRoute.includes(
+		STATES.runningState.id.toString()
+	);
+
 	return (
 		<ClayLayout.ContainerFluid view>
 			<ClayLayout.ContentRow>
-				<ClayLayout.ContentCol>
+				<ClayLayout.ContentCol expand>
 					<h1>
 						<a href={project.url} target="blank">
 							{project.name}
 						</a>
 					</h1>
 
-					<p>Git: {project.location}</p>
+					<ClayLayout.ContentRow>
+						<ClayLayout.ContentCol expand>
+							<p>Git: {project.location}</p>
+						</ClayLayout.ContentCol>
+
+						<ClayLayout.ContentCol>
+							<p>
+								{'Filter: '}
+
+								<ClayLabel
+									displayType={
+										STATES.byId[
+											currentRoute.charAt(
+												currentRoute.length - 1
+											)
+										].displayType
+									}
+								>
+									{
+										STATES.byId[
+											currentRoute.charAt(
+												currentRoute.length - 1
+											)
+										].name
+									}
+								</ClayLabel>
+							</p>
```

Comment:
I'm gonna move this selected state into the dropdown using the check mark. This will align us more with the lexicon patterns